lib/cog/chat/slack/connector.ex

```diff
@@ -323,6 +332,44 @@ defmodule Cog.Chat.Slack.Connector do
     end
   end
 
+  defp normalize_channel_id(:by_id, id = "C" <> _, slack) do
```

Comment:
But you'd be calling the function with `:by_name` as the first argument, then, right?